README.md

```diff
@@ -7,3 +7,16 @@ The goal is a real time creative writing collaboration 'game'.
 Each participant takes a turn to add a limited amount of text (ie: 100 words max) to write a short story. Each story will have a maximum of 10 writers and a minimum of 2 who can be anonymous or invited friends. Once each story is finished, it can be exported as a pdf or doc.
 
 This will build the back end of the app which can then be used to further develop front ends on different platforms.
+
+### Running the app
+You must have these or a later version installed:
+
+Node version 10.10 | npm version 6.4.1
```

Comment:
Nice - just to make it easier to read, could we update this to a bulleted list, like so:

* Node.js version 10.10
* npm version 6.4.1